api/tests/test_asset_status_model.py

```diff
@@ -0,0 +1,91 @@
+
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
+
+from ..models import Asset, AssetModelNumber, AssetStatus
+
+User = get_user_model()
+client = APIClient()
+
+
+class AssetStatusModelTest(TestCase):
+    """Tests for the Asset Status Model"""
+
+    def setUp(self):
+        self.test_assetmodel1 = AssetModelNumber(model_number="IMN50987")
+        self.test_assetmodel2 = AssetModelNumber(model_number="IMN50986")
+        self.test_assetmodel1.save()
+        self.test_assetmodel2.save()
+
+        self.normal_user = User.objects.create(
+            email='test@site.com', cohort=10,
+            slack_handle='@test_user', password='devpassword'
+        )
+
+        self.test_asset = Asset(
+            asset_code="IC001",
+            serial_number="SN001",
+            model_number=self.test_assetmodel1,
+            assigned_to=self.normal_user
+        )
+        self.test_asset.save()
+        self.get_asset = Asset.objects.get(asset_code="IC001")
```

Comment:
`self.get_asset` sounds like a method. But since it's a variable, let's rename it to sound less like an action.

Reply:
Done @alexkiura